src/main/java/ch/njol/skript/expressions/ExprNamed.java

```diff
@@ -97,15 +121,15 @@ public Object get(Object obj) {
 			}
 		});
 	}
-	
+
 	@Override
 	public Class<? extends Object> getReturnType() {
 		return getExpr().getReturnType() == InventoryType.class ? Inventory.class : ItemType.class;
```

Comment:
I don't believe this is safe for variables, or any other expression that would have Object.class as the return type. We might have an InventoryType type in a variable but would be returning ItemType as the return type here. Likely another issue we can fix while we're here :)